client/src/components/AuditAdmin/Progress/AuditBoardsTable.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { Button } from '@blueprintjs/core'
+import { Cell, Column } from 'react-table'
+
+import { Confirm, useConfirm } from '../../Atoms/Confirm'
+import { Table } from '../../Atoms/Table'
+
+interface IAuditBoardMinimal {
+  id: string
+  name: string
+  signedOffAt: string | null
+}
+
+interface IProps {
+  areAuditBoardsOnline: boolean
+  auditBoards: IAuditBoardMinimal[]
+  reopenAuditBoard: (auditBoard: IAuditBoardMinimal) => Promise<void>
+}
+
+const AuditBoardsTable = ({
+  areAuditBoardsOnline,
+  auditBoards,
+  reopenAuditBoard,
+}: IProps) => {
+  const { confirm, confirmProps } = useConfirm()
+
+  const columns: Column<IAuditBoardMinimal>[] = [
+    {
+      Header: 'Audit Board',
+      Cell: (cell: Cell<IAuditBoardMinimal>) => {
+        const auditBoard = cell.row.original
+        return auditBoard.name
+      },
+    },
+  ]
+  if (areAuditBoardsOnline) {
+    columns.push({
+      Header: 'Actions',
+      // eslint-disable-next-line react/display-name
+      Cell: (cell: Cell<IAuditBoardMinimal>) => {
+        const auditBoard = cell.row.original
+        return (
+          <Button
+            disabled={!auditBoard.signedOffAt}
+            onClick={() =>
+              confirm({
+                title: 'Confirm',
+                description: `Are you sure you want to reopen ${auditBoard.name}?`,
+                yesButtonLabel: 'Reopen',
+                onYesClick: () => reopenAuditBoard(auditBoard),
+              })
+            }
+          >
+            Reopen
+          </Button>
+        )
+      },
+    })
+  }
+  return (
+    <>
+      <Table columns={columns} data={auditBoards} />
```

Comment:
The `Table` component was an experiment with using the react-table library. I think my takeaway after trying it out is that the fanciness is maybe more trouble than it's worth. I can't remember exactly what made it hard, but I just remember that I kept running into annoying little problems and found code using `Table` to be tricky to maintain. Since then I've been leaning towards using the basic styled `HTMLTable` instead.

I probably should have left a comment to that effect to deprecate `Table`.

It's not a big deal either way but just wanted to clue you in on my thinking and hear your thoughts.

Reply:
Very helpful context! I did find the interface hard to understand, and I generally think that compositional approaches to tables (e.g. through multiple components, `table`, `th`, `tr`, `td`, etc.) are more intuitive

Reply:
https://github.com/votingworks/arlo/pull/1573/commits/34770e246afe016ab576bbb02528c02bbb88e7e1